MobileCenterDistribute/MobileCenterDistribute/MSDistributePrivate.h

```diff
@@ -1,8 +1,21 @@
 #import <Foundation/Foundation.h>
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wreserved-id-macro"
+#ifndef __IPHONE_11_0
+#define __IPHONE_11_0    110000
+#endif
+#pragma clang diagnostic pop
+
 #import "MSAlertController.h"
 #import "MSUIAppDelegate.h"
 #import "MSDistribute.h"
+#if __IPHONE_OS_VERSION_MAX_ALLOWED >= __IPHONE_11_0
```

Comment:
Any reasons for this?

Reply:
My 2 cents is SafariServices is weak linked so we should not have the need for this.